tests/unit/transforms/attachment-test.js

```diff
@@ -0,0 +1,62 @@
+import { moduleFor, test } from 'ember-qunit';
+
+import Ember from 'ember';
+
+let testSerializedData = {
+  'test.txt': {
+    content_type: 'text/plain',
+    data: 'hello world!'
+  },
+  'stub.json': {
+    stub: true,
+    content_type: 'application/json'
+  }
+};
+
+let testDeserializedData = [
+  Ember.Object.create({
+    name: 'test.txt',
+    content_type: 'text/plain',
+    data: 'hello world!'
+  }),
+  Ember.Object.create({
+    name: 'stub.json',
+    content_type: 'application/json',
+    stub: true
```

Comment:
Typically stubs also have `digest` and `length` fields. It makes me nervous not to check for that; does that mean we're not passing that data to relational-pouch?